### ISSUE TITLE
Android 8 or 9 : with most apps, the items array is empty

### ISSUE THREAD

**Issue by SimplyLab** [j3k0/cordova-plugin-openwith#54]
I can't understand why most of the time, the items array is empty when I share content.
Don't know how to investiguate

Object
  action: "SEND"
  exit: false
   items: Array(1)
      0: null
        length: 1__proto__: Array(0)__proto__: Object

**Comment by JochenHeizmann** [comment]
Same Problem here with Android 8. Haven't dived into it deeper, but it seems like the Serialization of the Intent doesn't work properly (in https://github.com/j3k0/cordova-plugin-openwith/blob/master/src/android/cc/fovea/openwith/OpenWithPlugin.java). Will try to dive deeper into it later on.

**Comment by JochenHeizmann** [comment]
@SimplyLab The plugin doesn't seem to support the text datatype on Android. If this is also the problem in your case, you can have a look at my fork with a fix: https://github.com/JochenHeizmann/cordova-plugin-openwith/commit/331e244a048c8aead1f58cc2e7e836893b0ddc55

(I haven't issued a pull request yet)

**Comment by niklas-e** [comment]
Yeah there seems to be problem with text content on Android (at least). The intent items are `null`. Have you tested your fixes @JochenHeizmann, could it be opened as PR?

**Comment by jj449** [comment]
I confirm this problem .  Still  exist. 
@JochenHeizmann  's   solution fix this , confirm .

**Comment by manix** [comment]
@JochenHeizmann the real mvp
